package-lock.json

```diff
@@ -24,7 +24,7 @@
       "integrity": "sha512-ZAR2bPHOl4Xg6eklUGpsdiIJ4+J1SNag1DHHrG/73Uz/nVwXqjgUtRPLoS+aVyieN9cSbc0E4LsU984tWcDyNg==",
       "dev": true,
       "requires": {
-        "@sinonjs/samsam": "^2 || ^3"
+        "@sinonjs/samsam": "2.1.3"
```

Comment:
I think npm install tries to install the latest dependencies that satisfies the versions in package.json, so that causes the lock file to change. AFAIK this is normal.

Reply:
In that case, why were the previous changes made (the one with `^2 || ^3"`, etc.)? Or should those types of changes only be made in package.json?

Reply:
I recently upgraded from npm 5 to npm 6, so it is possible that they changed the default behavior.

Reply:
I'm on npm 6.13.4. I think different npm versions generate the file slightly differently and that it's okay. But in CI we should be running `npm ci` so it installs deps using the lock file in that commit.

Reply:
So I ran npm install again and it undid all of the changes... I'm not exactly sure what happened here.